app/views/reports/index.html.erb

```diff
@@ -8,13 +8,11 @@
 </div>
 
 <hr>
+<!-- map -->
+<%= current_user.current_report %>
 
 <div class="container">
-  <% current_user.reports.each do |report| %>
-    <% if report.period.current? %>
-      <%= render report %>
-    <% end %>
-  <% end %>
+  <%= render "reports/report", report: current_user.reports.current_report.first %>
```

Comment:
Oh! Looks like you were already doing what I suggested. But yes, you can extract this to the `User#current_report` method, so you don't have to call `.first` on the scope everywhere and can simply do

```suggestion
  <%= render "reports/report", report: current_user.current_report %>
```